backend/src/main/scala/cromwell/backend/validation/ContinueOnReturnCodeValidation.scala

```diff
@@ -23,42 +21,29 @@ import scala.util.Try
   * `default` a validation with the default value specified by the reference.conf file.
   */
 object ContinueOnReturnCodeValidation {
-  lazy val instance: RuntimeAttributesValidation[ContinueOnReturnCode] = new ContinueOnReturnCodeValidation
-  def default(runtimeConfig: Option[Config]): RuntimeAttributesValidation[ContinueOnReturnCode] =
+  lazy val instance: RuntimeAttributesValidation[ReturnCode] = new ContinueOnReturnCodeValidation
+  def default(runtimeConfig: Option[Config]): RuntimeAttributesValidation[ReturnCode] =
     instance.withDefault(configDefaultWdlValue(runtimeConfig) getOrElse WomInteger(0))
   def configDefaultWdlValue(runtimeConfig: Option[Config]): Option[WomValue] =
     instance.configDefaultWomValue(runtimeConfig)
 }
 
-class ContinueOnReturnCodeValidation extends RuntimeAttributesValidation[ContinueOnReturnCode] {
+class ContinueOnReturnCodeValidation extends ReturnCodeValidation {
```

Comment:
I would love to see if we can collapse all the return code stuff into this validation, and have this be the last thing in the code path that's aware of the two different attributes. We could even continue calling it `ContinueOnReturnCode` to minimize the diff. This validation can look at both keys and make a decision about what `ContinueOnReturnCode` object to build. We can turn the `*` into a `ContinueOnReturnCodeFlag(true)`, which seems to have the same meaning.